Assets/MixedRealityToolkit/_Core/Definitions/Utilities/AxisType.cs

```diff
@@ -32,14 +32,14 @@ public enum AxisType
         /// <summary>
         /// Position only Axis analogue input.
         /// </summary>
-        ThreeDoFPosition,
+        ThreeDofPosition,
```

Comment:
why the casing changes?

Reply:
Thought I put this in the description. Must have missed that.

So that it renders properly in the inspector.

old: `Three Do F Position`
new: `Three Dof Position`